scipy/spatial/transform/rotation.py

```diff
@@ -861,7 +861,11 @@ def from_euler(cls, seq, angles, degrees=False):
                              "num_axes), got {}.".format(angles.shape))
 
         quat = _elementary_quat_compose(seq, angles, intrinsic)
-        return cls(quat[0] if is_single else quat, normalize=False, copy=False)
+
```

Comment:
The `quat` object above may be slightly different vs. the one Thomas patched on `master` because of `seq.encode()`